src/sentry/api/endpoints/broadcast_index.py

```diff
@@ -33,11 +33,14 @@ def _serialize_objects(self, items, request):
         serializer_cls = self._get_serializer(request)
         return serialize(items, request.user, serializer=serializer_cls())
 
-    def get(self, request):
+    def _secondary_filtering(self, request, organization_slug, queryset):
```

Comment:
Would be nice to see a comment here about this being used in the open source product.